### DISCUSSION TITLE
OpenRadioss in Google Colab

### DISCUSSION THREAD

**Discussion by saullocastro** [OpenRadioss/OpenRadioss#463] [Show and tell]
Hello everyone!

I have prepared a simple workflow to run OpenRadioss using Google Colab.

Google Colab's resources are not so bad after all, with the [available bird strike example](https://openradioss.atlassian.net/wiki/spaces/OPENRADIOSS/pages/8847361/Bird+Strike+On+Windshield) running in about 35 minutes.

The example can be found in the following gist:

https://colab.research.google.com/gist/saullocastro/9d10547dabbfb377dab00177d407554c

Enjoy it!

Best,
Saullo

**Comment by saullocastro** [comment]
To automate the post-processing using Python, I'm waiting until Altair releases [an output converter that will convert ](https://openradioss.atlassian.net/wiki/spaces/OPENRADIOSS/pages/21397510/Pre+and+Post+Processing+for+OpenRadioss)from .h3d or .Annn to vtk and csv, such that Paraview can be used.

**Reply by PaulAltair** [reply]
I believe converters are already in the latest builds since a couple of days ago? (Annn - vtk and T01 - csv)

**Reply by OlivierWienholtz** [reply]
source code is available on Git, 
Unfortunately, the tools are not yet built...

**Reply by Serince** [reply]
I think you can add following scripts to convert paraview file.
!wget https://raw.githubusercontent.com/OpenRadioss/OpenRadioss/main/tools/anim_to_vtk/src/anim_to_vtk.cpp
!wget https://raw.githubusercontent.com/OpenRadioss/OpenRadioss/main/tools/th_to_csv/src/th_to_csv.c
Compiling scripts:
!g++ -DLINUX -o anim_to_vtk anim_to_vtk.cpp
!gcc -g -o th_to_csv th_to_csv.c
Converting results
!./anim_to_vtk BIRD_WINDSHIELD_v1.h3d > BIRD.vtk
!./th_to_csv BIRD_WINDSHIELD_v1T01

**Reply by saullocastro** [reply]
This works indeed, thank you @Serince

**Reply by ehan08** [reply]
anim_to_vtk works well on both Annn and h3d file. Thanks.

**Comment by OlivierWienholtz** [comment]
Hello,
We have slighty modifed the directories & add build scripts.
Pls have a look at the latest Commits.

Have a nice Day
